source/set-mutable.d.ts

```diff
@@ -0,0 +1,35 @@
+import {Except} from './except';
+import {Mutable} from './mutable';
+
+/**
+Create a type that converts the given keys from `readonly` to mutable. The remaining keys are kept as is.
+
```

Comment:
While it's implied, think we should make it clear here and in `Mutable` that this does not affect the values, like `{foo: readonly string[]}`.